prog/weaver/main.go

```diff
@@ -307,11 +307,15 @@ func main() {
 			trackerName = "awsvpc"
 		}
 		var allContainerIDs []string
+		var runningContainerIDs []string
```

Comment:
Better to have these lists inside the scope they are used in.
If you look at 7075b5d9fd6872c647633e9ec9580e25753dc901 where `allContainerIDs` was introduced, the variable was hoisted so it could be used in two places.  But not you're removing that need so the code should go back more like it was before.
In the same vein, `runningContainerIDs` should be inside `findExistingAddresses`.